python/ray/serve/api.py

```diff
@@ -1712,47 +1731,76 @@ def from_yaml(cls, str_or_file: Union[str, TextIO]) -> "Application":
                 Serve YAML config files.
 
         Returns:
-            Application: a new application object containing the deployments.
+            Application: a new Application object containing the deployments.
+        """
+        return cls.from_dict(yaml.safe_load(str_or_file))
+
+    def get_ingress(self) -> Optional[Deployment]:
```

Comment:
can you update this to be:
```python
@property
def ingress(self):
    ...
```

to match the docstring and the `deployments` interface?

Reply:
I updated it.